lib/dotcom/schedule_finder/upcoming_departures.ex

```diff
@@ -136,15 +141,35 @@ defmodule Dotcom.ScheduleFinder.UpcomingDepartures do
           route_type: route_type,
           now: now
         }),
+      arrival_substatus:
+        arrival_substatus(%{
+          predicted_schedule: predicted_schedule,
+          route_type: route_type
+        }),
       headsign: trip.headsign,
+      platform_name: platform_name(predicted_schedule),
+      status: predicted_schedule |> PredictedSchedule.status(),
```

Comment:
Do you need this field? I'm not sure if you're using upcoming_departure.status directly (you read the same value into `arrival_substatus`)

Reply:
Oh oops - I added this for debugging and forgot to remove it.